OpenRA.Mods.Common/Traits/Attack/AttackFollow.cs

```diff
@@ -36,7 +40,7 @@ public class AttackFollow : AttackBase, INotifyOwnerChanged, IDisableAutoTarget,
 		public Target RequestedTarget { get; private set; }
 		public Target OpportunityTarget { get; private set; }
 
-		Mobile mobile;
+		public Mobile Mobile;
```

Comment:
Consider change this to private set property

Reply:
Refactored a bit so that making it public is no longer necessary.